Sming/Libraries/modbusino/README.rst

```diff
@@ -0,0 +1,27 @@
+modbusino RTU Library (modbus slave)
+====================================
+
+The author's project is at https://github.com/stephane/modbusino
+
+modbusino is lightweight RTU modbus slave library that supports 'read holding registers' and 'write multiple registers' functions.
+
+Four environment variables deifne the RS485 interface hardware pins and pre- and post-transmission delay.
```

Comment:
Replace `deifne` with `define`. Make sure to run a spell checker.

Reply:
To be honest my Eclipse's spell checker doesn't work. I'm investigating.